### DISCUSSION TITLE
can't login, says password incorrect.

### DISCUSSION THREAD

**Discussion by r0uted** [bagisto/bagisto#10316] [Q&A]
it shows message error "Please Check Your Credentials And Try Again"
Email and password

email:admin@example.com
password:admin123

I used docker instructions to deploy and I am not able to login with default admin credentials. 

Steps : 
```
docker pull webkul/bagisto:2.2.2
docker run -it -d -p 80:80 webkul/bagisto:2.2.2
```

I even tried [this](https://forums.bagisto.com/topic/545/how-to-change-your-admin-password-from-the-backend-when-you-can-t-recover-or-smtp-is-not-configured) method using backend, but still I am not able to access the website using updated password.

**Comment by suraj-webkul** [comment]
Hello, @r0uted

It seems that everything should be working fine with the default admin credentials. However, sometimes the issue can be related to the port or caching. You might want to try accessing the application using a different port.

Here's an alternative method to run the application:

1. **Stop the running container**:
```bash
   docker stop <container_id>
```

2. **Run the container with a different port**:
```bash
    docker run -it -d -p 81:80 webkul/bagisto:2.2.2
```

After that, try accessing the website again via `http://127.0.0.1:81`. This can help if there is a port conflict.

Let me know if that works for you!

**Reply by r0uted** [reply]
it didnt work.

**Reply by devansh-webkul** [reply]
May be in your case port is different.

**Comment by r0uted** [comment]
I tried: 
- changing ports 
- disabling cache 
- incognito window 
- different browser 

 and it still didnt work, says password incorrect. 

I feel it weird that even at first run on my browser, it says password incorrect. 

<img width="400" alt="image" src="https://github.com/user-attachments/assets/641dcf3c-ad56-46dd-a376-5553d165d221">


![telegram-cloud-photo-size-5-6174880859941881506-y](https://github.com/user-attachments/assets/80100a95-4aec-4778-8cbf-fbc63b211d95)

**Reply by r0uted** [reply]
@suraj-webkul can it be any help of you please ? 

Thanks.

**Reply by devansh-webkul** [reply]
This is customer login page. You need to register the user first.
